blender/LilySurfaceScrapper/Scrappers/Cc0texturesScrapper.py

```diff
@@ -34,51 +36,44 @@ def canHandleUrl(cls, url):
     def fetchVariantList(self, url):
         """Get a list of available variants.
         The list may be empty, and must be None in case of error."""
-        html = self.fetchHtml(url)
-        if html is None:
+        if Cc0texturesScrapper._texture_cache is None:
+            Cc0texturesScrapper._texture_cache = self.fetchXml("https://cc0textures.com/api/getDownloadLinks.php")
+
+        root = Cc0texturesScrapper._texture_cache
+
+        texture = root.find(f"assets/item[link='{url}']")
```

Comment:
We might need to normalize the url here, because the copy-pasted one might include or not some trailing slash `/` or some query string like `?fbclick=blablabla`.

Reply:
Is this implementation acceptable? Or will you tackle this issue yourself?
```suggestion
        parsed_url = urlparse(url.strip('/'))
        query = parse_qs(parsed_url.query)
        parsed_url = parsed_url._replace(query=urlencode({ 'tex': query.get('tex', None) }, True))

        texture = root.find(f"assets/item[link='{parsed_url.geturl()}']")
```

Reply:
This is what I was thinking about :) We'll see if there are some edge cases that are not supported but I thnik that it'll be good!